tmt/utils.py

```diff
@@ -1652,11 +1652,17 @@ def render_exception(exception: BaseException) -> List[str]:
     """ Render the exception and its causes for printing """
 
     def _indent(iterable: Iterable[str]) -> List[str]:
-        return [
-            f'{INDENT * " "}{line}'
-            for item in iterable
-            for line in item.splitlines()
-            ]
+        lines: List[str] = []
+
+        for item in iterable:
+            if not item:
+                lines.append(item)
+
```

Comment:
Just one minor question: there is one empty line (L1660), any special purpose?

Reply:
Well, yes, it's a way to visually separate logical steps in the code :) In this case, two branches of an `if` condition block. There are more empty lines, 1656 and 1664, BTW, serving the same purpose: "declare list" -> "loop over items in `iterable`" -> "do something if `item` is empty" -> "do something if `item` is not empty" -> "return collected lines". They do not have any functional purpose, just visual aid.

Reply:
Cool, thanks very much for explanation :-)